src/oas2/transformers/getExamplesFromSchema.ts

```diff
@@ -3,6 +3,8 @@ import { isObject } from 'lodash';
 import { Schema } from 'swagger-schema-official';
 
 export function getExamplesFromSchema(data: Schema & { 'x-examples'?: Dictionary<unknown> }): Dictionary<unknown> {
+  if (!isObject(data)) return {};
```

Comment:
Hmm, I saw this merged but I got a review request so just posting this for the posterity.

This is wrong to me somehow. The `Schema` type is already implying that `data` is an object and not undefined.

If this line is required it means somebody calling this function is lying/casting. That's not this function's problem; it should not be the its job of this function to validate the passed value.